### PR TITLE
Add deeplab v3 image segmentation model

### PR DESCRIPTION
**Pull request by Roulbac**
### Description

- [X] I've included a concise description of what each node does

### Styleguide

- [X] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [ ] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

Screenshot
Screen Shot 2023-06-20 at 10 26 27 AM

### CODE REVIEW

File: MANIFEST/deeplab_v3.manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+COMMAND:
+  - name: DeepLabV3 Image Segmentation
+    key: DEEPLAB_V3
+    type: SEGMENTATION
```
Comment (scorinaldi) [inline MANIFEST/deeplab_v3.manifest.yaml:4]: please add a newline

**Review by scorinaldi — CHANGES_REQUESTED** [review]
This looks good! I had only one small cosmetic change.

Also, since this node is so complicated, could you include either a screenshot of the output of your example so we know it worked properly, or (ideally) a working unit test? See https://github.com/flojoy-io/nodes/blob/c28808f2d0aec8f116cdb3fccb46c4f5256574e9/TRANSFORMERS/ARITHMETIC/ADD/ADD_test_.py for an example of a unit test

File: MANIFEST/deeplab_v3.manifest.yaml (hunk 1 above)
Comment (Roulbac) [reply MANIFEST/deeplab_v3.manifest.yaml:4]: Not sure if you want it at the head or tail, please use "Add a suggestion"

**Comment by Roulbac** [comment]
@scorinaldi ptal

**Review by scorinaldi — APPROVED** [review]
LGTM! modulo a small suggested change

File: MANIFEST/deeplab_v3.manifest.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Roulbac) [reply MANIFEST/deeplab_v3.manifest.yaml:4]:
```suggestion
    type: SEGMENTATION
    
```

**Pull request merged**